_posts/2021-04-17-dd_00001_intro.md

```diff
@@ -0,0 +1,324 @@
+header:
+  overlay_color: "#000"
+  overlay_filter: "0.5"
+  overlay_image: /assets/images/digital_design.jpg
+---
+
+Welcome to our first post on **Digital Design**! We are glad that you found your
+way into this very exciting topic. In this post, we will try to give you an
+overview of the various aspects of digital design. For each topic, we will of
+course make a detailed and comprehensive post. This series intends to share our
+knowledge and experience in designing digital systems with you. Unlike many
+other resources, we will start from the ground up and explain the basic
+principles behind many of the technologies in digital design in a simple way.
+So, if you've never heard of digital design before, this is the right place to
+start! And, if you are already an experienced student or engineer, you are also
+welcome, of course; maybe you might learn something new along the way. We also
+invite you (newbies + experienced) to give us feedback and ideas for future
+posts!
+
+Now, let's jump in, shall we?
+
+If you look around you, you will probably find at least one _electronic_ device
+near you. In fact, you are probably reading this post through one of those
+devices. Have you ever wondered, how such small devices which surround us every
+day, can do so much powerful things? For most people, their devices are just
+made up of "black magic", and they "just work". But if you have made it until
+this point, you are probably interested in _what_ that "black magic" actually
+_is_, right? Then keep on reading and we will shed some light on this complex,
+but fascinating topic!
+
+## Microchips
+
+All of our intelligent devices are powered by what are called "**microchips**".
+You probably have heard that term before. A microchip (or just, _chip_) is a
+very tiny **electronic circuit** that can be integrated into other systems, to
+do some useful tasks for that system. There are a variety of chips designed
+specifically for certain tasks. For example:
+
+- Measuring some state (e.g, temperature) of the system
+- Controlling the system
+- Processing data aquired by the system
+- ...
+
+<!-- ![microchips](/assets/images/dd-00001-microchips.jpg){: .align-center} -->
+
+<figure style="width:85%" class="align-center">
+  <img src="/assets/images/dd-00001-microchips.jpg" alt="">
+  <!-- <figcaption class="figure-caption text-center">Microchips</figcaption> -->
+</figure>
+
+Usually, a complex technical system consolidates multiple (10s to 100s) of
+different chips. Just look at a modern car: Nowadays, you can also think of a
+car as a supercomputer on four wheels. In a car, there are hundreds of different
+chips built into it. For example, there are chips that control the correct
+operation of the engine, others control the air conditioning and still others
+take care of the infotainment system. And things don't stop here: with emerging
+technologies, such as autonomous driving, future cars will require even more,
+and powerful chips.
```

Comment:
do you mean 'more and more powerful' or 'more powerful'

Reply:
8d2ab27 fixed